frontend/src/common/queries/collections.ts

```diff
@@ -182,17 +173,27 @@ function fetchCollection() {
 
     let publishedCounterpart;
 
+    // If the collection is a revision, find the corresponding published
+    // collection for diff'ing purposes.
     if (collection.revision_of) {
       const publicCollectionURL = apiTemplateToUrl(API_URL + API.COLLECTION, {
         id: collection.revision_of,
       });
-      response = await fetch(publicCollectionURL, DEFAULT_FETCH_OPTIONS);
-      json = await response.json();
+      const publishedCounterpartResponse = await fetch(
+        publicCollectionURL,
+        DEFAULT_FETCH_OPTIONS
+      );
+      const publishedCounterpartJSON =
+        await publishedCounterpartResponse.json();
```

Comment:
Renamed variables here to clearly indicated they are associated with the published counterpart and not the collection (related to `let` to `const` update above).